packages/wonder-blocks-tooltip/components/tooltip-bubble.js

```diff
@@ -98,8 +94,10 @@ export default class TooltipBubble extends React.Component<Props, State> {
                 id={id}
                 role="tooltip"
                 data-placement={placement}
-                onMouseEnter={() => this._handleMouseEnter()}
-                onMouseLeave={() => this._handleMouseLeave()}
+                // eslint-disable-next-line react/jsx-handler-names
```

Comment:
I'd suggest to remove this comment and rename that function to something that eslint could accept. Maybe something like `this.handleMouseEnter`?